dynatrace-aws-log-forwarder-template.yaml

```diff
@@ -48,6 +48,9 @@ Parameters:
     # https://aws.amazon.com/blogs/compute/query-for-the-latest-amazon-linux-ami-ids-using-aws-systems-manager-parameter-store/
     Type: 'AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>'
     Default: '/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2'
+  LogForwarderSetupName:
```

Comment:
I think additional param is unnecessary
I would propose something like this:
$ACCOUNT_ID:$REGION:$STACK_NAME 
e.g. "412345678:us-east1:dynatrace-log-forwarder"
AccountID and region are probably accessible from Kinesis incoming record, but I think it would be safer to take all 3 from cloudformation stack data. 
I think cloudformation should pass env variable (probably with already formed string like I suggested) to lambda and then lambda should add it to metadata

Reply:
Thank you for your comment. I will investigate it.

Reply:
As we discussed it will be in the next commit :-)